Ginger/GingerCoreNET/ActionsLib/ActExcel.cs

```diff
@@ -507,5 +510,36 @@ private string CalculatedPrimaryKeyFilter(DataRow dataRow)
             string pk = GetInputParamCalculatedValue(nameof(PrimaryKeyColumn)).Replace("`", "");
             return dataRow[pk].GetType() == typeof(int) ? pk + "=" + dataRow[pk].ToString() : pk + "=" + "'" + dataRow[pk].ToString() + "'";
         }
+
+        private static void ValidateDataToWrite(string CellWriteValue)
+        {
+            if (string.IsNullOrEmpty(CellWriteValue))
+            {
+                throw new ArgumentException("Data To Write Value Error: Please Enter a value to be written in the cell");
+            }
+            if (CellWriteValue.StartsWith("'"))
+            {
+                int index = CellWriteValue.LastIndexOf("'");
+                if (index <= 0)
+                {
+                    throw new ArgumentException("Data To Write Value Error: Please Enter appropriate value to be written in the cell");
+                }
+                bool isNumber = CellWriteValue.Substring(1, index - 1).All(char.IsDigit);
```

Comment:
It will throw exception when user has configured '{Var name=xyz}' and variable is digit.
Please check what is changed to break it and there we can fix the issue